src/FrozenClock.php

```diff
@@ -8,7 +8,7 @@
 
 final class FrozenClock implements Clock
 {
-    public function __construct(private DateTimeImmutable $now)
+    public function __construct(private DateTimeImmutable $now = new DateTimeImmutable())
```

Comment:
What's the need for this?

I mean, we already have `FrozenClock::fromUTC()` for the "I just want a Clock for tests" use case.